roles/openshift_master_facts/tasks/main.yml

```diff
@@ -30,10 +30,9 @@
       embedded_etcd: "{{ openshift_master_embedded_etcd | default(None) }}"
       embedded_kube: "{{ openshift_master_embedded_kube | default(None) }}"
       embedded_dns: "{{ openshift_master_embedded_dns | default(None) }}"
-      dns_port: "{{ openshift_master_dns_port | default(None) }}"
+      dns_port: "{{ openshift_master_dns_port | default('53' if not openshift.common.version_gte_3_2_or_1_2 else None ) }}"
```

Comment:
We probably want to set this is as a default openshift_facts based on the whether dnsmasq is configured rather than using the version fact here.